### PR TITLE
Update to beta6

### PR DESCRIPTION
**Pull request by WilliamBZA**
Update the container to Beta6

Ping @Particular/container-maintainers

### CODE REVIEW

File: src/NServiceBus.CastleWindsor.AcceptanceTests/app.config [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (danielmarbach) [inline src/NServiceBus.CastleWindsor.AcceptanceTests/app.config]: Sure this is required?

File: src/NServiceBus.CastleWindsor.Tests/APIApprovals.Approve.approved.txt [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-    public class WindsorBuilder : NServiceBus.Container.ContainerDefinition
-    {
-        public WindsorBuilder() { }
-        public override NServiceBus.ObjectBuilder.Common.IContainer CreateContainer(NServiceBus.Settings.ReadOnlySettings settings) { }
-    }
-    public class static WindsorExtensions
-    {
-        public static void ExistingContainer(this NServiceBus.Container.ContainerCustomizations customizations, Castle.Windsor.IWindsorContainer container) { }
-    }
+﻿[assembly: System.CLSCompliantAttribute(true)]
```
Comment (danielmarbach) [inline src/NServiceBus.CastleWindsor.Tests/APIApprovals.Approve.approved.txt]: Is this just line endings? Hard to follow

**Comment by danielmarbach** [comment]
@WilliamBZA two comments

File: src/NServiceBus.CastleWindsor.AcceptanceTests/app.config (hunk 1 above)
Comment (WilliamBZA) [inline src/NServiceBus.CastleWindsor.AcceptanceTests/app.config]: Nope, sorry. Will remove.

File: src/NServiceBus.CastleWindsor.Tests/APIApprovals.Approve.approved.txt (hunk 2 above)
Comment (WilliamBZA) [inline src/NServiceBus.CastleWindsor.Tests/APIApprovals.Approve.approved.txt]: Yeah I'm not sure. This shouldn't have been a change. Will revert.

**Comment by WilliamBZA** [comment]
Updated to beta 7 since it was out.

**Comment by andreasohlund** [comment]
@WilliamBZA rc1 is out, want me to update this?

**Comment by WilliamBZA** [comment]
Yes please. If there's no objections from the other @Particular/container-maintainers I think I'll merge it then.

**Comment by danielmarbach** [comment]
Not from my side

**Comment by andreasohlund** [comment]
Changes included in #16
